stingray/modeling/parameterestimation.py

```diff
@@ -285,8 +285,10 @@ def print_summary(self, lpost, log=None):
             if par in parnames:
                 idx = parnames.index(par)
 
-                log.info("{:<20.5f} +/- {:<20.5f} ".format(self.p_opt[idx],
-                                                  self.err[idx]))
+                err_info = " (no error estimate)"
+                if self.err is not None:
+                    err_info = " +/- {:<20.5f}".format(self.err[idx])
+                log.info("{:<20.5f}{} ".format(self.p_opt[idx], err_info))
```

Comment:
This is an unexpected change given what this PR does. Can you explain why this is happening here?

Reply:
It's a bug I found when testing without statsmodels installed. It's an optional dependency, so it was good practice to test without it, but we were not doing it and we never realized there was a bug :)
It's a small fix anyway. If statsmodels is not installed, self.err is None.